dev-client/src/translations/es.json

```diff
@@ -715,9 +715,12 @@
             "required_data_title": "Requisitos para la recogida de datos"
         },
         "required_method": "{{method}} (obligatorio)",
+        "collection_method_summary": {
+            "soilTexture": "Clase de textura"
+        },
         "collection_method": {
             "slope": "Incluye la inclinación, forma, aspecto y posición del pendiente del suelo.",
-            "soilTexture": "Incluye clase de textura, porcentaje de arcilla y fragmentos de roca",
+            "soilTexture": "Textura del suelo",
```

Comment:
question: is there work tracked somewhere to also update slope in a similar way? or is it not a problem because of how they're displayed?